d2/src/main/java/com/linkedin/d2/jmx/RelativeLoadBalancerStrategyJmx.java

```diff
@@ -0,0 +1,168 @@
+package com.linkedin.d2.jmx;
+
+import com.linkedin.d2.balancer.clients.TrackerClient;
+import com.linkedin.d2.balancer.strategies.LoadBalancerQuarantine;
+import com.linkedin.d2.balancer.strategies.relative.RelativeLoadBalancerStrategy;
+import com.linkedin.d2.balancer.strategies.relative.TrackerClientState;
+import com.linkedin.d2.balancer.util.partitions.DefaultPartitionAccessor;
+import com.linkedin.util.degrader.CallTracker;
+import java.net.URI;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+
+public class RelativeLoadBalancerStrategyJmx implements RelativeLoadBalancerStrategyJmxMBean
+{
+  private final RelativeLoadBalancerStrategy _strategy;
+
+  public RelativeLoadBalancerStrategyJmx(RelativeLoadBalancerStrategy strategy)
+  {
+    _strategy = strategy;
+  }
+
+  @Override
+  public double getLatencyStandardDeviation()
+  {
+    Map<TrackerClient, TrackerClientState> stateMap =
+        _strategy.getPartitionState(DefaultPartitionAccessor.DEFAULT_PARTITION_ID).getTrackerClientStateMap();
+
+    List<Double> weightedLatencies = stateMap.keySet().stream()
+        .map(trackerClient -> getWeightedLatency(trackerClient.getLatestCallStats()))
+        .collect(Collectors.toList());
+
+    double avgLatency = weightedLatencies.stream()
+        .mapToDouble(Double::doubleValue)
+        .average()
```

Comment:
Is there a way to get the average of all requests in stead of the average of all hosts? If host A gets 100 requests and host B gets 10 requests, we probably want (100 * latencyA + 10 * latencyB) / 110 instead of (latencyA + latencyB) / 2. That will be consistent to how to calculate the cluster average

Reply:
BTW you can get avgClusterLatency from PartitionStats in PartitionData, we can probably see if we can reuse PartitionStats, because it's also used for metrics emitting purpose

Reply:
> BTW you can get avgClusterLatency from PartitionStats in PartitionData, we can probably see if we can reuse PartitionStats, because it's also used for metrics emitting purpose

I've thought of using PartitionStats. But it only keeps the cluster level latency/call count/error rate. If I want to find out the call stats for each individual host, I still have to go through the `TrackerClient` map. So I'm thinking using the call stats from the map directly would produce a more accurate result.

Reply:
OK I will leave this implementation decision to you, as long as we are using the correct way to calculate avg cluster latency :)

Reply:
Fixed in the new commit